src/balances/index.ts

```diff
@@ -5,6 +5,7 @@ const getTokenBalance = async (
   walletAddress: string,
   token: Token
 ): Promise<TokenAmount | null> => {
+  debugger
```

Comment:
Do we really want this?

Reply:
This was removed loooong before your review :P